applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -1279,6 +1467,93 @@ static int cmd_program_info(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+#define FILE_LIST_BUF_SIZE 512
+static int cmd_file_list(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	char buf[FILE_LIST_BUF_SIZE];
+	size_t buf_size = FILE_LIST_BUF_SIZE;
+	char *dir_path = NULL;
+
+	if (argc > 2) {
```

Comment:
Could we rather check 
```suggestion
	if (argc != 2) {
```
print error and return? 

As it stands now, if argc == 1, dir_path will be NULL.

Reply:
This is the intended way - if `sd_card_list_files` receives a `NULL` path it'll print out the contents of the root.